microbench/src/main/java/io/netty/handler/codec/http2/Http2PseudoHeadersLookupBenchmark.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2023 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Copyright 2015 Twitter, Inc.
```

Comment:
why this header ?

Reply:
I belvive Is my GitHub copilot!!! Ahahahah just I don't understand why twitter, as well...